Automattic/wp-calypso
# Signup: cleanup new user survey experiment

Related to #

## Proposed Changes

* This PR reverts the changes introduced in #87995 for the new user survey.

## Why are these changes being made?


* pau2Xa-605-p2

## Testing Instructions



* Since the new step affected only the onboarding flow, only testing `/start` is required. Ensure that you can complete the flow.

## Pre-merge Checklist
